js/utility/mDDuration.js

```diff
@@ -0,0 +1,7 @@
+export default function calculateDuration(startDate) {
+  const endDate = new Date().getTime();
```

Comment:
prefer `Date.now()` over the `Date` constructor.

// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now

```suggestion
  const now = Date.now();
```

Reply:
The Date.now() method was introduced in ECMAScript 5 and is not supported in Internet Explorer 8, which only supports up to ECMAScript 3. So if IE 8 compatibility is a concern we should use new Date().getTime() at the cost of slightly diminished code readability. [little discussion](https://stackoverflow.com/questions/33184096/date-new-date-date-valueof-vs-date-now)

Reply:
God bless Internet Explorer 8, is dead. 😅